client/me/concierge/main.js

```diff
@@ -98,15 +118,20 @@ export class ConciergeMain extends Component {
 
 	render() {
 		const { analyticsPath, analyticsTitle, site } = this.props;
+		const { reauthRequired } = this.state;
 		const siteId = site && site.ID;
-
 		return (
 			<Main>
 				<PageViewTracker path={ analyticsPath } title={ analyticsTitle } />
-				<QueryUserSettings />
-				<QuerySites />
-				{ siteId && <QueryConciergeInitial siteId={ siteId } /> }
-				{ siteId && <QuerySitePlans siteId={ siteId } /> }
+				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
+				{ reauthRequired && (
```

Comment:
This prevents the whole form from showing for me. If I understand its meaning correctly, shouldn't it be `! reauthRequired`?

Reply:
Good catch. Result of thrashing around too much to work around the failing test, and then landing on a bad patch.